daffodil-runtime1/src/main/scala/org/apache/daffodil/udf/UserDefinedFunctionService.scala

```diff
@@ -28,7 +28,6 @@ import java.lang.reflect.Method
 import org.apache.daffodil.dpath.NodeInfo
 import java.io.Serializable
 import java.io.ObjectInputStream
-import scala.language.existentials
```

Comment:
Had to add a language feature flag for 2.11 due to the import being seen as unused in 2.12
  - See https://github.com/scala/bug/issues/11467